lib/seek/assets_common.rb

```diff
@@ -8,6 +8,11 @@ module AssetsCommon
     include Seek::AssetsStandardControllerActions
 
     def find_display_asset(asset = eval("@#{controller_name.singularize}"))
+      if !asset.versioned?
+        # No version, the display_ is the main asset, non-versionned
+        eval "@display_#{asset.class.name.underscore} = asset"
```

Comment:
generally `eval` should be avoided, even though there is no obvious security flaw in this case.

You could use `instance_variable_set("@display_#{asset.class.name.underscore}", asset)` instead.

But also see my above comment about compatibility for non-versioned things

Reply:
Changed it anyway, also the 2nd one which was there before :)